### ISSUE TITLE
URL Queue

### ISSUE THREAD

**Issue by joshmtucker** [JadenGeller/Helium#22]
Hey Jaden! Great job with Helium. As we discussed on Twitter, I think it would be awesome if there was some way to queue URLs so you could have videos progress back-to-back. Cheers!

**Comment by archshift** [comment]
Wouldn't that require some kind of awareness by Helium about Youtube's video player?

Perhaps a hotkey or something to advance to the next URL?

**Comment by joshmtucker** [comment]
@archshift Possibly (in the context of YouTube). I was thinking more in a general sense. You may want to watch a set of videos from a number of different TV shows (as an example), so it would be cool to be able to throw them into a queue so that it can pop in the next URL and go.

**Comment by alesf** [comment]
Or if you give Helium a playlist to play it could know what to play next

**Comment by joshmtucker** [comment]
@alesf Yep, same premise.

**Comment by hye** [comment]
Or maybe the website should done that job,
How a browser tell `when` to move on to next url?

**Comment by JadenGeller** [comment]
I kind of agree with @hye here after some thought. Unless someone has a good reason (and mechanism) for it to behave otherwise, I'll probably close this issue.

**Comment by joshmtucker** [comment]
@JadenGeller Makes sense – wasn't aware of whether you could detect those sort of things easily. I'll put my thumbs up for the Favorite/Recents drawer that was proposed :). A way to expand on that too is to have a static set of services – have YouTube, Hulu, Netflix, Vimeo, etc. always in said menu so they are quickly accessible.

**Comment by distantnative** [comment]
I'd love to have a queue. Doesn't need to auto-switch to the next video, cause that'd be hard to detect. Just with a button "play next", so I can throw in a bunch of URLs in the beginning and not always keep them open in Safari in some tabs to remember them.
